core/classes/FlowRoundRobinAssigner.cls

```diff
@@ -10,6 +10,8 @@ global without sharing class FlowRoundRobinAssigner {
     global String assignmentRepoClassName;
     @InvocableVariable(label='Query To Retrieve Possible Assignees')
     global String queryToRetrieveAssignees;
+    @InvocableVariable(label='Alternative to Assignee Query - Collection of Users from Flow')
+    global List<User> collectionOfAssignees = new List<User>();
```

Comment:
Because it's possible for ownership for some SObjects to be for a Queue as well as for Users (and because, theoretically, you could also be using this to round robin more than just ownership), I think it would be nice to keep this a bit more generic (can still default to Id, I think, so long as this also supports the use of the other optional property that stipulates the field name to pull per record).

Reply:
See, I'm not sure about keeping it generic. It's so that it can be an easy plug and play for an Admin without them having to think hard about what they're doing. Maybe having some types of validation (via either CPE or apex) to ensure that it's only Queue/Group/User and have it as List/Set<Id>?

Reply:
I respect that, thanks for giving your side of things. Let me think about it for a bit - I'll probably end up approving these changes as is tomorrow